code/modules/holodeck/area_copy.dm

```diff
@@ -52,6 +54,11 @@ GLOBAL_LIST_INIT(duplicate_forbidden_vars,list(
 				contained_atom.flags_1 |= HOLOGRAM_1
 			if(M.circuit)
 				M.circuit.flags_1 |= HOLOGRAM_1
+
+	if(ismob(O))	//Overlays are carried over despite disallowing them, if a fix is found remove this.
+		var/mob/M = O
+		M.cut_overlays()
+		M.regenerate_icons()
```

Comment:
Attempted to disallow overlays in duplicate_forbidden_vars, but it wasnt working and copied them over anyway. If it hadnt, would likely need a regenerate_icons regardless. Suggestions welcome.